src/BMS.cpp

```diff
@@ -109,15 +111,15 @@ void BMS::initializationErrorState() {
     if (stateChanged) {
         bmsOK.writePin(BMS_NOT_OK);
         stateChanged = false;
-        clearVoltageReadings();
+        //clearVoltageReadings();
```

Comment:
NIT: Shouldn't leave commented out code. If you are planning to come back to it, the best option is to add a little explanation in the comments above the line